gym/envs/robotics/robot_env.py

```diff
@@ -70,6 +70,10 @@ def seed(self, seed=None):
         return [seed]
 
     def step(self, action):
+        assert (
+            np.shape(action)[-1] == self.action_space.shape[-1]
```

Comment:
Would it be better if we do `assert np.array(action).shape == self.action_space.shape`? I would also try to be more explicit about this, raising a `ValueError` instead of an assertion error.

Reply:
Great, I think raising a `ValueError` may be a better choice.